package.json

```diff
@@ -98,7 +98,7 @@
   },
   "dependencies": {
     "babel-runtime": "^6.11.6",
-    "js-worker-search": "^1.4.0",
+    "js-worker-search": "^1.4.1",
```

Comment:
This shouldn't really be necessary. 1.4.1 will already be installed if the semver range is ^1.4.0.